File: tools/bench/answers/python_haiku/s1/04_binary.py
numbers = [13, 64]
for num in numbers:
    binary = ""
    n = num
    if n == 0:
        binary = "0"
    else:
        while n > 0:
            binary = str(n % 2) + binary
            n = n // 2
    print(binary)

File: tools/bench/answers/go_haiku/s1/04_binary.go
package main

import "fmt"

func main() {
	nums := []int{13, 64}
	for _, num := range nums {
		if num == 0 {
			fmt.Println("0")
		} else {
			var binary string
			n := num
			for n > 0 {
				binary = string(rune('0' + n%2)) + binary
				n /= 2
			}
			fmt.Println(binary)
		}
	}
}
